server/index.js

```diff
@@ -1,24 +1,52 @@
-const express = require("express");
-const mongoose = require("mongoose");
-
-const compile = require("./routes/api/compile");
-const zip = require("./routes/api/zip");
-const generate = require("./routes/api/generate");
+const express = require('express');
+const mongoose = require('mongoose');
 
+const compile = require('./routes/api/compile');
+const zip = require('./routes/api/zip');
+const generate = require('./routes/api/generate');
+const upload = require('express-fileupload');
 const app = express();
-
+var fs = require('fs');
 //Database config goes here
 
 //Database connection goes here
-
-app.get("/", (req, res) => {
-  res.send("Hello World");
+app.use(upload());
+app.get('/', (req, res) => {
+	res.send('Hello World');
+});
+app.get('/fileupload', function(req, res) {
```

Comment:
Just a few more changes.
```suggestion
app.get('/upload-file', function(req, res) {
```